scripts/update-drafts.ts

```diff
@@ -1,8 +1,8 @@
 import { Compat } from "compute-baseline/browser-compat-data";
 import fs from "node:fs/promises";
 import { fileURLToPath } from "node:url";
+import {Document} from "yaml";
```

Comment:
There are some whitespace differences to surrounding code, but that's fine. I'll add this file to the Prettier config after this which will reformat stuff for consistency.